Firestore/Example/Tests/Util/FSTDispatchQueueTests.mm

```diff
@@ -216,10 +216,12 @@ - (void)testEnterCheckedOperationDisallowsNesting {
 - (void)testCanScheduleCallbacksInTheFuture {
   _expectation = [self expectationWithDescription:@"Expected steps"];
   _expectedSteps = @[ @1, @2, @3, @4 ];
-  [_queue dispatchAsync:[self blockForStep:1]];
-  [_queue dispatchAfterDelay:0.005 timerID:timerID1 block:[self blockForStep:4]];
-  [_queue dispatchAfterDelay:0.001 timerID:timerID2 block:[self blockForStep:3]];
-  [_queue dispatchAsync:[self blockForStep:2]];
+  [_queue dispatchAsync:^{
```

Comment:
For `dispatchAfterDelay`, it is necessary because it assumes it's being run on the queue. However, for the others, it's unnecessary, so I turned them back into `dispatchAsync` calls and moved them out.